docs/en/ingest-management/elastic-agent/elastic-agent-encryption.asciidoc

```diff
@@ -0,0 +1,29 @@
+[elastic-agent-encryption]
+= Elastic Agent Configuration Encryption
+
+It is important for users to understand the security model of Elastic Agent and how it handles sensitive values in integration configurations.
+At a high level, Elastic Agent receives configuration data from Fleet Server over an encrypted connection, persists those secrets in an encrypted form on disk.
+This persistence is necessary to allow agents to continue to operate, even in the event of a loss of connectivity to the Fleet server.
+
+The entirety of the Fleet Agent Policy is encrypted at rest, but is recoverable if the user has access to both the encrypted configuration data and the associated key.
```

Comment:
Can we double-confirm this @cmacknz ?

Reply:
@aleksmaus would be good to have you review this as well.